projects/storefrontstyles/scss/header/_header.scss

```diff
@@ -0,0 +1,156 @@
+@import 'theme';
```

Comment:
It wont work unless you use relative paths

Reply:
Let me correct that, it will work as part of our build process but not for externals. It wont work for customers because they don't add the styles lib to their `"stylePreprocessorOptions"/""includePaths`.

I think it's better that way because by using relative imports we aren't dependent to the angular build system and it makes the styles lib more independent.